src/ui/components/componentmanager.js

```diff
@@ -200,6 +200,10 @@ export default class ComponentManager {
     return this._activeComponents.find(c => c.constructor.type === type);
   }
 
+  getActiveComponents (type) {
```

Comment:
For my edification how is this being used? I didn't see any usages in the fresh code

Reply:
Ah ok I see it in the hitchhiker-theme PR now